tests/refs.test.js

```diff
@@ -0,0 +1,138 @@
+import { t as tw, equal } from "twist"
+import { JSDOM } from "jsdom"
+import { h, patch } from "../index.js"
+
+// we enhance twist's original function `t` 
```

Comment:
Why do we need generators to test this?

Reply:
I guess this may have been a misunderstanding on my side.
I was of the impression that in `twist` "one test" means "one invocation of a twist assert function", so this refs test file would contain a bit more than 40 tests ... and to be count properly in the summary I had to `yield` each single one of them.
Normally I would say this refs test file contains only three tests (with over 40 assertions).
Please show me a few lines on code how this is to be done with `twist` normally - thanks - and I will fix it.

Reply:
No worries! If we merge this feature, I'll take care of the tests myself. 🙆‍♂️